docs/how-to-guides/image-processing/bundle-adjustment-in-isis.md

```diff
@@ -73,19 +89,51 @@ An explanation of the configuration options passed to autoseed is as follows:
 
 ## Step 6: Performing Bundle Adjustment with Jigsaw
 
-Finally, with all the prerequisite information generated and attached, it is possible to perform bundle adjustment.  Within ISIS, this is performed using the `jigsaw` utility. `jigsaw` is highly configurable, and it has many options that can be viewed [here](https://isis.astrogeology.usgs.gov/8.1.0/Application/presentation/Tabbed/jigsaw/jigsaw.html).  For the purposes of this tutorial, configuration is left to a minimum.
+Finally, with all the prerequisite information generated and attached, it is possible to perform bundle adjustment.  Within ISIS, this is performed using the `jigsaw` utility. `jigsaw` is highly configurable, and it has many options that can be viewed [here](https://isis.astrogeology.usgs.gov/8.1.0/Application/presentation/Tabbed/jigsaw/jigsaw.html). 
+
+### Command Line Usage
+
+#### Option 1: For images without a CSM State String
+If you chose option 1 (spiceinit) for step 2, then you should choose this option for jigsaw.  A basic command line invocation for jigsaw is as follows:
 
 ```console
 jigsaw fromlist=image_list.lis cnet=control_network.net onet=control_network_jig.net update=no file_prefix=jig maxits=10 camsolve=angles camera_angles_sigma=.25
 ```
 
-An explanation of the configuration options passed to jigsaw is as follows:
+#### Option 2: For images with a CSM State String
+If you chose option 2 (csminit) for step 2, then you must also choose this option for jigsaw. When using jigsaw to bundle adjust a `csminit`-ed image, the user must pass CSM-specific parameter names via the `csmsolvelist` parameter.  Allowable values and their descriptions are as follows.
+
+| Parameter | Description |
+|---|---|
+| IT Pos. Bias     |  "In Track Position Bias" - a constant shift in the spacecraft's position parallel to the flight path |
```

Comment:
It looks like units are needed for all of these. Also, consider telling folks where are correlated. For example, solving for CT Pos. Bias and Phi.

Reply:
Good call on adding units, but I'm at a loss here. csminit.xml shows that all parameters are "m," and the code shows that getParameterUnits() is hard-coded to "m" with a note that "All parameters are meters or scaled to meters."  Does that seem correct...?